packages/bpk-tokens/tokens/android/base.raw.json

```diff
@@ -854,6 +860,24 @@
       },
       "name": "SPACING_XXL"
     },
+    "FONT_FAMILY": {
+      "value": "\"sans-serif\"",
```

Comment:
This is a bit of a hack as this just gets serialized in Javascript we need the quotes to make sure it ends up as a stirng in JS

Reply:
I think when I was doing this originally I used `"'sans-serif'"` to achieve the same thing